subprojects/language-java/build.gradle.kts

```diff
@@ -61,6 +61,8 @@ dependencies {
         because("ProjectBuilder test (JavaLanguagePluginTest) loads services from a Gradle distribution.")
     }
 
+    // TODO: Make these available for all integration tests? Maybe all tests?
```

Comment:
Do you have an answer for this todo? Why do we need the jetbrains annotations in `languageJava`?

Reply:
`@Language` comes from them.

I don't have a good answer. In general I am against using the JetBrains annotations widely, because they introduce their own `@Nullable`/`@NotNull` etc. annotations. But then again, having them present in only some subprojects is also problematic. So no, I have no answer.

The intention here is to make the build work with Spock 2.2, though, that does not include the transitive dependency on JetBrains annotations anymore.

Reply:
👍